### PR TITLE
Chore: fix scheduler code comment typos

### PR DESCRIPTION
**Pull request by Jerry-yz**
#### What type of PR is this?

/kind cleanup

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by Jerry-yz** [comment]
> #### What type of PR is this?
/sig scheduling
> #### What this PR does / why we need it:
> #### Which issue(s) this PR fixes:
> Fixes #
> 
> #### Special notes for your reviewer:
> #### Does this PR introduce a user-facing change?
> ```
> NONE
> ```
> 
> #### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:
> ```
> 
> ```

File: pkg/scheduler/framework/interface.go [hunk 1]
```diff
@@ -363,7 +363,7 @@ type QueueSortPlugin interface {
 // Failures from other extension points are regarded as temporal errors (e.g., network failure),
 // and the scheduler requeue Pods without this extension point - always requeue Pods to activeQ after backoff.
 // This is because such temporal errors cannot be resolved by specific cluster events,
-// and we have no choise but keep retrying scheduling until the failure is resolved.
```
Comment (kerthcet) [inline pkg/scheduler/framework/interface.go:366]:
```suggestion
// and we have no choice but keep retrying scheduling until the failure is resolved.
```

**Review by kerthcet — COMMENTED** [review]
/approve
/kind cleanup
/ok-to-test

**Comment by kerthcet** [comment]
/retitle Chore: fix scheduler code comment typos

**Pull request merged**